apps/server/docker-compose.dev.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.15.1
```

Comment:
Needs to be ES 7, ES8 is not compatible with Arranger.

Reply:
added a separate docker-compose file to start elasticsearch7 (`apps/server/docker-compose-es7.dev.yml`) or es8 (`apps/server/docker-compose-es8.dev.yml`)
This will let the dev choose which one to use.